Automattic/woocommerce-services
# Setup shipping zones with live rate methods based on Setup Wizard choices

To test:

* Check out https://github.com/Automattic/woocommerce-connect-server/tree/add/shipping-zone-setup-from-3-2-wizard
* Set one or both of the following options to `1`:
  * `woocommerce_setup_domestic_live_rates_zone`
  * `woocommerce_setup_intl_live_rates_zone`
* Verify that the corresponding zone has USPS or Canada Post (based on store country) method setup up with defaults
